targets/bmv2/test/run_test_batch.py

```diff
@@ -0,0 +1,39 @@
+import subprocess
+import os
+
+def generate_config(seed, p4_program):
+    command = f"backends/p4tools/modules/p4rtsmith/p4rtsmith --target bmv2 --arch v1model --seed {seed} {p4_program}"
```

Comment:
Do not hardcode these paths, instead use a python Pathlib variable for them. For example this seems like it should be a parameter to the p4rtsmith binary.